diff -up ./Project/GNU/Library/configure.ac.0000 ./Project/GNU/Library/configure.ac
--- ./Project/GNU/Library/configure.ac.0000	2014-09-06 08:23:57.017068352 +0300
+++ ./Project/GNU/Library/configure.ac	2014-09-06 08:26:31.780627802 +0300
@@ -29,6 +29,7 @@ AC_DISABLE_SHARED
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
 AC_PROG_INSTALL
+PKG_PROG_PKG_CONFIG
 
 dnl #########################################################################
 dnl Configure
